### PR TITLE
(phi0588)_all_files (eventually)

### PR DESCRIPTION
**Pull request by AlisonBabeu**
phi0588.phi001 and phi0588.phi002

Updated cts_.xml and headers.
Slightly edited bibliographic metadata and then added in page level links.

hi @lcerrato I just wanted to make sure the way I changed these two files was ok before I go through with the rests, its nice to have all the works in one edition for once.

### CODE REVIEW

File: data/phi0588/abo001/__cts__.xml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi0588.abo001" xml:lang="lat" groupUrn="urn:cts:latinLit:phi0588">
             <ti:title xml:lang="lat">Miltiades</ti:title>
             <ti:edition urn="urn:cts:latinLit:phi0588.abo001.perseus-lat2" workUrn="urn:cts:latinLit:phi0588.abo001">
-              <ti:label xml:lang="eng">Miltiades, Cornelii Nepotis Vitae</ti:label>
-              <ti:description xml:lang="eng">Nepos, Cornelius, creator; Fleckeisen, Alfred, 1820-1899, editor; Halm, Karl, 1809-1882, original editor</ti:description>
+              <ti:label xml:lang="lat">Miltiades</ti:label>
+              <ti:description xml:lang="mul">Nepos, Cornelius. Cornelii Nepotis Vitae. Fleckeisen, Alfred; Halm, Karl, editors.
```
Comment (lcerrato) [inline data/phi0588/abo001/__cts__.xml:5]: We generally have common name here, so "Cornelius Nepos." versus "Nepos, Cornelius."

File: data/phi0588/abo001/phi0588.abo001.perseus-lat2.xml [hunk 2]
```diff
@@ -73,6 +75,7 @@ schematypens="http://purl.oclc.org/dsdl/schematron"?>
     </profileDesc>
     
     <revisionDesc>
+      <change when="2023-01-18" who="Alison Babeu">Updated header and added in page level link</change>
```
Comment (lcerrato) [inline data/phi0588/abo001/phi0588.abo001.perseus-lat2.xml:78]: A few minor header changes don't require the change log update. Feel free to skip this if you don't want the extra work.

File: data/phi0588/abo002/__cts__.xml [hunk 3]
```diff
@@ -1,8 +1,9 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi0588.abo002" xml:lang="lat" groupUrn="urn:cts:latinLit:phi0588">
             <ti:title xml:lang="lat">Themistocles</ti:title>
             <ti:edition urn="urn:cts:latinLit:phi0588.abo002.perseus-lat2" workUrn="urn:cts:latinLit:phi0588.abo002">
-              <ti:label xml:lang="eng">Themistocles, Cornelii Nepotis Vitae</ti:label>
-              <ti:description xml:lang="eng">Nepos, Cornelius, creator; Fleckeisen, Alfred, 1820-1899, editor; Halm, Karl, 1809-1882, original editor</ti:description>
+              <ti:label xml:lang="lat">Themistocles, Cornelii Nepotis Vitae</ti:label>
```
Comment (lcerrato) [inline data/phi0588/abo002/__cts__.xml:4]: Inconsistent with first file, which dropped the "Cornelii Nepotis Vitae"

File: data/phi0588/abo002/__cts__.xml [hunk 4]
```diff
@@ -1,8 +1,9 @@
...
+              <ti:description xml:lang="mul">Nepos, Cornelius. Cornelii Nepotis Vitae. Fleckeisen, Alfred; Halm, Karl, editors.
```
Comment (lcerrato) [inline data/phi0588/abo002/__cts__.xml:5]: Same as above, Cornelius Nepos preferred.

File: data/phi0588/abo001/__cts__.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/phi0588/abo001/__cts__.xml:5]: Ok, I will do that!

File: data/phi0588/abo001/phi0588.abo001.perseus-lat2.xml (hunk 2 above)
Comment (AlisonBabeu) [reply data/phi0588/abo001/phi0588.abo001.perseus-lat2.xml:78]: Okely dokely!

File: data/phi0588/abo002/__cts__.xml (hunk 3 above)
Comment (AlisonBabeu) [reply data/phi0588/abo002/__cts__.xml:4]: Dang, inconsistent with only two files, normally it takes me up to three!  But thanks I will make these changes and proceed!

**Review by lcerrato — APPROVED** [review]

**Pull request merged**